dev-python/termcolor/termcolor-2.2.0.ebuild

```diff
@@ -6,15 +6,13 @@ EAPI=8
 DISTUTILS_USE_PEP517=hatchling
 PYTHON_COMPAT=( python3_{9..11} pypy3 )
 
-inherit distutils-r1
+inherit distutils-r1 pypi
 
 DESCRIPTION="ANSI Color formatting for output in terminal"
 HOMEPAGE="
 	https://github.com/termcolor/termcolor/
 	https://pypi.org/project/termcolor/
 "
-# rename is for avoiding conflict with dev-cpp/termcolor
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz -> python-${P}.tar.gz"
```

Comment:
This one I'm not really sure about — but then I suppose we can restore the rename when we actually hit conflicts.

Reply:
I think the rename is needed for termcolor, since this looks like too generic name
I'm also afraid of hitting file conflict with downloaded file of dev-cpp/termcolor that remain in some distdir